sdk/lib/crt/stdlib/stdlib.cmake

```diff
@@ -1,11 +1,13 @@
 
 list(APPEND LIBCNTPR_STDLIB_SOURCE
+    stdlib/rand_nt.c
     stdlib/qsort.c
```

Comment:
qsort should be before rand_nt when sorting the file list in alphabetical order.